<?xml version="1.0" encoding="UTF-8"?><testrun duration="1181502" footerText="Generated by PyCharm on 25.07.2023, 11:32" name="pytest for test_smoke.TestSmoke">
    <count name="total" value="49"/>
    <count name="failed" value="8"/>
    <count name="passed" value="41"/>
    <config nameIsGenerated="true" configId="tests" name="pytest for test_smoke.TestSmoke">
        <module name="lime-qa"/>
        <option name="INTERPRETER_OPTIONS" value=""/>
        <option name="PARENT_ENVS" value="true"/>
        <option name="SDK_HOME" value=""/>
        <option name="WORKING_DIRECTORY" value="C:/Users/Qasquad1/PycharmProjects/lime-qa/tests"/>
        <option name="IS_MODULE_SDK" value="true"/>
        <option name="ADD_CONTENT_ROOTS" value="true"/>
        <option name="ADD_SOURCE_ROOTS" value="true"/>
        <option name="_new_keywords" value="&quot;&quot;"/>
        <option name="_new_parameters" value="&quot;&quot;"/>
        <option name="_new_additionalArguments" value="&quot;&quot;"/>
        <option name="_new_target" value="&quot;test_smoke.TestSmoke&quot;"/>
        <option name="_new_targetType" value="&quot;PYTHON&quot;"/>
        <method v="2"/>
    </config>
    <suite duration="1181502" locationUrl="python&lt;C:\Users\Qasquad1\PycharmProjects\lime-qa\tests&gt;://tests" name="tests" status="failed">
        <suite duration="1181502" locationUrl="python&lt;C:\Users\Qasquad1\PycharmProjects\lime-qa\tests&gt;://tests.test_smoke" name="test_smoke" status="failed">
            <suite duration="1181502" locationUrl="python&lt;C:\Users\Qasquad1\PycharmProjects\lime-qa\tests&gt;://tests.test_smoke.TestSmoke" name="TestSmoke" status="failed">
                <test duration="28632" locationUrl="python&lt;C:\Users\Qasquad1\PycharmProjects\lime-qa\tests&gt;://tests.test_smoke.TestSmoke.test_pay_bonus_card_not_valid_self" name="test_pay_bonus_card_not_valid_self" metainfo="test_pay_bonus_card_not_valid_self" status="passed">
                    <output type="stdout">PASSED      [  2%]
</output>
                </test>
                <test duration="35346" locationUrl="python&lt;C:\Users\Qasquad1\PycharmProjects\lime-qa\tests&gt;://tests.test_smoke.TestSmoke.test_pay_bonus_card_no_many_self" name="test_pay_bonus_card_no_many_self" metainfo="test_pay_bonus_card_no_many_self" status="passed">
                    <output type="stdout">PASSED        [  4%]
</output>
                </test>
                <test duration="4842" locationUrl="python&lt;C:\Users\Qasquad1\PycharmProjects\lime-qa\tests&gt;://tests.test_smoke.TestSmoke.test_main_screen_main_menu_lk" name="test_main_screen_main_menu_lk" metainfo="test_main_screen_main_menu_lk" status="passed">
                    <output type="stdout">PASSED           [  6%]
</output>
                </test>
                <test duration="12252" locationUrl="python&lt;C:\Users\Qasquad1\PycharmProjects\lime-qa\tests&gt;://tests.test_smoke.TestSmoke.test_registration" name="test_registration" metainfo="test_registration" status="passed">
                    <output type="stdout">PASSED                       [  8%]
</output>
                </test>
                <test duration="24657" locationUrl="python&lt;C:\Users\Qasquad1\PycharmProjects\lime-qa\tests&gt;://tests.test_smoke.TestSmoke.test_cart_product" name="test_cart_product" metainfo="test_cart_product" status="passed">
                    <output type="stdout">PASSED                       [ 10%]
</output>
                </test>
                <test duration="48761" locationUrl="python&lt;C:\Users\Qasquad1\PycharmProjects\lime-qa\tests&gt;://tests.test_smoke.TestSmoke.test_add_del_fav_user" name="test_add_del_fav_user" metainfo="test_add_del_fav_user" status="passed">
                    <output type="stdout">PASSED                   [ 12%]
</output>
                </test>
                <test duration="25286" locationUrl="python&lt;C:\Users\Qasquad1\PycharmProjects\lime-qa\tests&gt;://tests.test_smoke.TestSmoke.test_fav_card_product_check" name="test_fav_card_product_check" metainfo="test_fav_card_product_check" status="passed">
                    <output type="stdout">PASSED             [ 14%]
</output>
                </test>
                <test duration="6776" locationUrl="python&lt;C:\Users\Qasquad1\PycharmProjects\lime-qa\tests&gt;://tests.test_smoke.TestSmoke.test_main_screen_main_menu_search" name="test_main_screen_main_menu_search" metainfo="test_main_screen_main_menu_search" status="passed">
                    <output type="stdout">PASSED       [ 16%]платье миди из шелка и вискозы
платье миди с без рукавов
платье миди с без рукавов
платье мини без рукавов
платье кроше в стиле бохо
платье из искусственной кожи в стиле гранж
платье миди без рукавов
платье миди ажурной вязки с бахромой
платье миди ажурной вязки с бахромой
платье мини с открытыми плечами
платье миди без рукавов
платье миди ажурной вязки с бахромой
платье мини с этническим принтом
платье миди с оборками
платье мини с кристаллами
платье с открытыми плечами и чокером
платье с открытой линией плеч
платье-рубашка изо льна
платье из вискозы с градиентным принтом
платье с отложным воротником и лентами
платье мини с градиентным принтом
платье из искусственной кожи с эффектом потертости
платье асимметричного кроя из искусственной кожи
платье из атласа с бахромой
платье мини с цветочным принтом
платье-бюстье из искусственной кожи
платье мини с ярусными оборками
платье мини с оборками на манжетах
платье-бюстье из искусственной кожи
платье-рубашка изо льна
платье мини с оборками
платье миди с вырезом
платье с разрезами со шнуровкой
платье из искусственной кожи с декоративным цветком
платье-рубашка в длине мини
платье мини с объемными рукавами
платье мини в бельевом стиле
платье с открытой спиной и контрастным съемным галстуком
платье мини с кристаллами
платье с чокером-цветком
</output>
                </test>
                <test duration="25810" locationUrl="python&lt;C:\Users\Qasquad1\PycharmProjects\lime-qa\tests&gt;://tests.test_smoke.TestSmoke.test_payment_cart_no_valid_promo" name="test_payment_cart_no_valid_promo" metainfo="test_payment_cart_no_valid_promo" status="passed">
                    <output type="stdout">PASSED        [ 18%]
</output>
                </test>
                <test duration="10839" locationUrl="python&lt;C:\Users\Qasquad1\PycharmProjects\lime-qa\tests&gt;://tests.test_smoke.TestSmoke.test_cart_product_discount" name="test_cart_product_discount" metainfo="test_cart_product_discount" status="passed">
                    <output type="stdout">PASSED              [ 20%]
</output>
                </test>
                <test duration="101750" locationUrl="python&lt;C:\Users\Qasquad1\PycharmProjects\lime-qa\tests&gt;://tests.test_smoke.TestSmoke.test_main_screen_menu" name="test_main_screen_menu" metainfo="test_main_screen_menu" status="failed">
                    <output type="stdout">FAILED                   [ 22%] переход выполнен не в каталог
</output>
                    <output type="stderr">tests\test_smoke.py:102 (TestSmoke.test_main_screen_menu)test_smoke.py:109: in test_main_screen_menu
    page.catalog_menu_parents_link_clothes()
..\pages\smoke.py:324: in catalog_menu_parents_link_clothes
    self.assert_check_coincidence("catalog", url_catalog, " переход выполнен не в каталог")
..\pages\base.py:83: in assert_check_coincidence
    assert expression1 in expression2, print(allureText)
E   AssertionError: None
</output>
                </test>
                <test duration="4377" locationUrl="python&lt;C:\Users\Qasquad1\PycharmProjects\lime-qa\tests&gt;://tests.test_smoke.TestSmoke.test_login_negative" name="test_login_negative" metainfo="test_login_negative" status="passed">
                    <output type="stdout">PASSED                     [ 24%]
</output>
                </test>
                <test duration="31491" locationUrl="python&lt;C:\Users\Qasquad1\PycharmProjects\lime-qa\tests&gt;://tests.test_smoke.TestSmoke.test_pay_card_no_many_pick_point" name="test_pay_card_no_many_pick_point" metainfo="test_pay_card_no_many_pick_point" status="passed">
                    <output type="stdout">PASSED        [ 26%]
</output>
                </test>
                <test duration="12376" locationUrl="python&lt;C:\Users\Qasquad1\PycharmProjects\lime-qa\tests&gt;://tests.test_smoke.TestSmoke.test_main_screen_main_menu_basket" name="test_main_screen_main_menu_basket" metainfo="test_main_screen_main_menu_basket" status="passed">
                    <output type="stdout">PASSED       [ 28%]
</output>
                </test>
                <test duration="10287" locationUrl="python&lt;C:\Users\Qasquad1\PycharmProjects\lime-qa\tests&gt;://tests.test_smoke.TestSmoke.test_cart_add_product_user" name="test_cart_add_product_user" metainfo="test_cart_add_product_user" status="passed">
                    <output type="stdout">PASSED              [ 30%]
</output>
                </test>
                <test duration="10861" locationUrl="python&lt;C:\Users\Qasquad1\PycharmProjects\lime-qa\tests&gt;://tests.test_smoke.TestSmoke.test_search" name="test_search" metainfo="test_search" status="failed">
                    <output type="stdout">FAILED                             [ 32%]солнцезащитные очки
 значение артикула не соответствует
</output>
                    <output type="stderr">tests\test_smoke.py:500 (TestSmoke.test_search)test_smoke.py:508: in test_search
    page.search_successful_article()
..\pages\smoke.py:675: in search_successful_article
    self.assert_check_expressions(article, "1418-888", " значение артикула не соответствует")
..\pages\base.py:75: in assert_check_expressions
    assert expression1 == expression2, print(allureText)
E   AssertionError: None
</output>
                </test>
                <test duration="46499" locationUrl="python&lt;C:\Users\Qasquad1\PycharmProjects\lime-qa\tests&gt;://tests.test_smoke.TestSmoke.test_bonus_card_pick_point" name="test_bonus_card_pick_point" metainfo="test_bonus_card_pick_point" status="failed">
                    <output type="stdout">FAILED              [ 34%]
</output>
                    <output type="stderr">tests\test_smoke.py:826 (TestSmoke.test_bonus_card_pick_point)..\venv\lib\site-packages\selene\core\wait.py:102: in for_
    return fn(self._entity)
..\venv\lib\site-packages\selene\core\condition.py:136: in __call__
    return self._fn(*args, **kwargs)
..\venv\lib\site-packages\selene\core\condition.py:91: in fn
    if not predicate(entity):
..\venv\lib\site-packages\selene\core\match.py:37: in &lt;lambda&gt;
    'is visible', lambda element: element().is_displayed()
..\venv\lib\site-packages\selene\core\entity.py:179: in __call__
    return self._locator()
..\venv\lib\site-packages\selene\core\locator.py:34: in __call__
    return self._locate()
..\venv\lib\site-packages\selene\core\entity.py:1529: in &lt;lambda&gt;
    f'{self}.element({by})', lambda: self.driver.find_element(*by)
..\venv\lib\site-packages\selenium\webdriver\remote\webdriver.py:976: in find_element
    return self.execute(Command.FIND_ELEMENT, {
..\venv\lib\site-packages\selenium\webdriver\remote\webdriver.py:321: in execute
    self.error_handler.check_response(response)
..\venv\lib\site-packages\selenium\webdriver\remote\errorhandler.py:242: in check_response
    raise exception_class(message, screen, stacktrace)
E   selenium.common.exceptions.NoSuchElementException: Message: no such element: Unable to locate element: {"method":"xpath","selector":"//div[contains(text(),'Спасибо!')]"}
E     (Session info: chrome=114.0.5735.248)
During handling of the above exception, another exception occurred:
test_smoke.py:839: in test_bonus_card_pick_point
    page.pay_order_without_3d()
..\pages\payment.py:591: in pay_order_without_3d
    self.wait_element_assure(self.title_thank_you_page_text)
..\pages\base.py:65: in wait_element_assure
    element.assure(visible, 15)
..\venv\lib\site-packages\selene\core\entity.py:627: in assure
    return self.should(condition, timeout)
..\venv\lib\site-packages\selene\core\entity.py:538: in should
    return self.with_(Config(timeout=timeout)).should(condition)
..\venv\lib\site-packages\selene\core\entity.py:540: in should
    super().should(condition)
..\venv\lib\site-packages\selene\core\entity.py:127: in should
    self.wait.for_(condition)
..\venv\lib\site-packages\selene\core\wait.py:125: in for_
    raise self._hook_failure(failure)
E   selene.core.exceptions.TimeoutException: Message: 
E   
E   Timed out after 15s, while waiting for:
E   browser.element(('xpath', "//div[contains(text(),'Спасибо!')]")).is visible
E   
E   Reason: NoSuchElementException: Message: no such element: Unable to locate element: {"method":"xpath","selector":"//div[contains(text(),'Спасибо!')]"}
E     (Session info: chrome=114.0.5735.248)
E   
E   Screenshot: file://C:\Users\Qasquad1\.selene\screenshots\1690257105839\1690257105842.png
E   PageSource: file://C:\Users\Qasquad1\.selene\screenshots\1690257105839\1690257105842.html
</output>
                </test>
                <test duration="591" locationUrl="python&lt;C:\Users\Qasquad1\PycharmProjects\lime-qa\tests&gt;://tests.test_smoke.TestSmoke.test_main_screen_main_menu_fav" name="test_main_screen_main_menu_fav" metainfo="test_main_screen_main_menu_fav" status="passed">
                    <output type="stdout">PASSED          [ 36%]
</output>
                </test>
                <test duration="25627" locationUrl="python&lt;C:\Users\Qasquad1\PycharmProjects\lime-qa\tests&gt;://tests.test_smoke.TestSmoke.test_pay_card_not_valid_pick_point" name="test_pay_card_not_valid_pick_point" metainfo="test_pay_card_not_valid_pick_point" status="passed">
                    <output type="stdout">PASSED      [ 38%]
</output>
                </test>
                <test duration="15093" locationUrl="python&lt;C:\Users\Qasquad1\PycharmProjects\lime-qa\tests&gt;://tests.test_smoke.TestSmoke.test_cart_add_product_anonim" name="test_cart_add_product_anonim" metainfo="test_cart_add_product_anonim" status="passed">
                    <output type="stdout">PASSED            [ 40%]
</output>
                </test>
                <test duration="11226" locationUrl="python&lt;C:\Users\Qasquad1\PycharmProjects\lime-qa\tests&gt;://tests.test_smoke.TestSmoke.test_burger_menu_check_products_sections" name="test_burger_menu_check_products_sections" metainfo="test_burger_menu_check_products_sections" status="passed">
                    <output type="stdout">PASSED [ 42%]
</output>
                </test>
                <test duration="25392" locationUrl="python&lt;C:\Users\Qasquad1\PycharmProjects\lime-qa\tests&gt;://tests.test_smoke.TestSmoke.test_pay_receiving_courier" name="test_pay_receiving_courier" metainfo="test_pay_receiving_courier" status="passed">
                    <output type="stdout">PASSED              [ 44%]
</output>
                </test>
                <test duration="37156" locationUrl="python&lt;C:\Users\Qasquad1\PycharmProjects\lime-qa\tests&gt;://tests.test_smoke.TestSmoke.test_pay_card_pick_point" name="test_pay_card_pick_point" metainfo="test_pay_card_pick_point" status="passed">
                    <output type="stdout">PASSED                [ 46%]
</output>
                </test>
                <test duration="2944" locationUrl="python&lt;C:\Users\Qasquad1\PycharmProjects\lime-qa\tests&gt;://tests.test_smoke.TestSmoke.test_burger_menu_product_card" name="test_burger_menu_product_card" metainfo="test_burger_menu_product_card" status="passed">
                    <output type="stdout">PASSED           [ 48%]
</output>
                </test>
                <test duration="1789" locationUrl="python&lt;C:\Users\Qasquad1\PycharmProjects\lime-qa\tests&gt;://tests.test_smoke.TestSmoke.test_login" name="test_login" metainfo="test_login" status="passed">
                    <output type="stdout">PASSED                              [ 51%]
</output>
                </test>
                <test duration="34134" locationUrl="python&lt;C:\Users\Qasquad1\PycharmProjects\lime-qa\tests&gt;://tests.test_smoke.TestSmoke.test_payment_cart_order_more_6000" name="test_payment_cart_order_more_6000" metainfo="test_payment_cart_order_more_6000" status="failed">
                    <output type="stdout">FAILED       [ 53%]
</output>
                    <output type="stderr">tests\test_smoke.py:546 (TestSmoke.test_payment_cart_order_more_6000)..\venv\lib\site-packages\selene\core\wait.py:102: in for_
    return fn(self._entity)
..\venv\lib\site-packages\selene\core\condition.py:136: in __call__
    return self._fn(*args, **kwargs)
..\venv\lib\site-packages\selene\core\condition.py:91: in fn
    if not predicate(entity):
..\venv\lib\site-packages\selene\core\match.py:37: in &lt;lambda&gt;
    'is visible', lambda element: element().is_displayed()
..\venv\lib\site-packages\selene\core\entity.py:179: in __call__
    return self._locator()
..\venv\lib\site-packages\selene\core\locator.py:34: in __call__
    return self._locate()
..\venv\lib\site-packages\selene\core\entity.py:1529: in &lt;lambda&gt;
    f'{self}.element({by})', lambda: self.driver.find_element(*by)
..\venv\lib\site-packages\selenium\webdriver\remote\webdriver.py:976: in find_element
    return self.execute(Command.FIND_ELEMENT, {
..\venv\lib\site-packages\selenium\webdriver\remote\webdriver.py:321: in execute
    self.error_handler.check_response(response)
..\venv\lib\site-packages\selenium\webdriver\remote\errorhandler.py:242: in check_response
    raise exception_class(message, screen, stacktrace)
E   selenium.common.exceptions.NoSuchElementException: Message: no such element: Unable to locate element: {"method":"xpath","selector":"//div[contains(text(), " Скидка на заказ")]"}
E     (Session info: chrome=114.0.5735.248)
During handling of the above exception, another exception occurred:
test_smoke.py:557: in test_payment_cart_order_more_6000
    page.sum_order_with_discount()
..\pages\payment.py:221: in sum_order_with_discount
    self.wait_element(self.discount_string) #ошибка - нет скидки в корзине при заказе более 6000
..\pages\base.py:57: in wait_element
    browser.s(locator).should(be.visible)
..\venv\lib\site-packages\selene\core\entity.py:540: in should
    super().should(condition)
..\venv\lib\site-packages\selene\core\entity.py:127: in should
    self.wait.for_(condition)
..\venv\lib\site-packages\selene\core\wait.py:125: in for_
    raise self._hook_failure(failure)
E   selene.core.exceptions.TimeoutException: Message: 
E   
E   Timed out after 10s, while waiting for:
E   browser.element(('xpath', '//div[contains(text(), " Скидка на заказ")]')).is visible
E   
E   Reason: NoSuchElementException: Message: no such element: Unable to locate element: {"method":"xpath","selector":"//div[contains(text(), " Скидка на заказ")]"}
E     (Session info: chrome=114.0.5735.248)
E   
E   Screenshot: file://C:\Users\Qasquad1\.selene\screenshots\1690257105839\1690257105843.png
E   PageSource: file://C:\Users\Qasquad1\.selene\screenshots\1690257105839\1690257105843.html
</output>
                </test>
                <test duration="13" locationUrl="python&lt;C:\Users\Qasquad1\PycharmProjects\lime-qa\tests&gt;://tests.test_smoke.TestSmoke.test_main_screen_logo" name="test_main_screen_logo" metainfo="test_main_screen_logo" status="passed">
                    <output type="stdout">PASSED                   [ 55%]
</output>
                </test>
                <test duration="1313" locationUrl="python&lt;C:\Users\Qasquad1\PycharmProjects\lime-qa\tests&gt;://tests.test_smoke.TestSmoke.test_burger_menu_bonus_card" name="test_burger_menu_bonus_card" metainfo="test_burger_menu_bonus_card" status="passed">
                    <output type="stdout">PASSED             [ 57%]
</output>
                </test>
                <test duration="27143" locationUrl="python&lt;C:\Users\Qasquad1\PycharmProjects\lime-qa\tests&gt;://tests.test_smoke.TestSmoke.test_card_self_no_valid_card" name="test_card_self_no_valid_card" metainfo="test_card_self_no_valid_card" status="passed">
                    <output type="stdout">PASSED            [ 59%]
</output>
                </test>
                <test duration="30280" locationUrl="python&lt;C:\Users\Qasquad1\PycharmProjects\lime-qa\tests&gt;://tests.test_smoke.TestSmoke.test_payment_cart_valid_promo" name="test_payment_cart_valid_promo" metainfo="test_payment_cart_valid_promo" status="passed">
                    <output type="stdout">PASSED           [ 61%]
</output>
                </test>
                <test duration="36947" locationUrl="python&lt;C:\Users\Qasquad1\PycharmProjects\lime-qa\tests&gt;://tests.test_smoke.TestSmoke.test_profile_order" name="test_profile_order" metainfo="test_profile_order" status="failed">
                    <output type="stdout">FAILED                      [ 63%] оформленный заказ отсутсвует в личном кабинете
</output>
                    <output type="stderr">tests\test_smoke.py:462 (TestSmoke.test_profile_order)test_smoke.py:497: in test_profile_order
    page.assert_check_expressions(number_order, number_order_lk, " оформленный заказ отсутсвует в личном кабинете")
..\pages\base.py:75: in assert_check_expressions
    assert expression1 == expression2, print(allureText)
E   AssertionError: None
</output>
                </test>
                <test duration="46695" locationUrl="python&lt;C:\Users\Qasquad1\PycharmProjects\lime-qa\tests&gt;://tests.test_smoke.TestSmoke.test_card_bonus_card_self_promo" name="test_card_bonus_card_self_promo" metainfo="test_card_bonus_card_self_promo" status="passed">
                    <output type="stdout">PASSED         [ 65%]
</output>
                </test>
                <test duration="23800" locationUrl="python&lt;C:\Users\Qasquad1\PycharmProjects\lime-qa\tests&gt;://tests.test_smoke.TestSmoke.test_pay_bonus_card_not_valid" name="test_pay_bonus_card_not_valid" metainfo="test_pay_bonus_card_not_valid" status="passed">
                    <output type="stdout">PASSED           [ 67%]
</output>
                </test>
                <test duration="30222" locationUrl="python&lt;C:\Users\Qasquad1\PycharmProjects\lime-qa\tests&gt;://tests.test_smoke.TestSmoke.test_pay_bonus_card_no_many_pick_point" name="test_pay_bonus_card_no_many_pick_point" metainfo="test_pay_bonus_card_no_many_pick_point" status="passed">
                    <output type="stdout">PASSED  [ 69%]
</output>
                </test>
                <test duration="7608" locationUrl="python&lt;C:\Users\Qasquad1\PycharmProjects\lime-qa\tests&gt;://tests.test_smoke.TestSmoke.test_add_fav_anonim" name="test_add_fav_anonim" metainfo="test_add_fav_anonim" status="passed">
                    <output type="stdout">PASSED                     [ 71%]
</output>
                </test>
                <test duration="30002" locationUrl="python&lt;C:\Users\Qasquad1\PycharmProjects\lime-qa\tests&gt;://tests.test_smoke.TestSmoke.test_bonus_card_no_many" name="test_bonus_card_no_many" metainfo="test_bonus_card_no_many" status="passed">
                    <output type="stdout">PASSED                 [ 73%]
</output>
                </test>
                <test duration="34633" locationUrl="python&lt;C:\Users\Qasquad1\PycharmProjects\lime-qa\tests&gt;://tests.test_smoke.TestSmoke.test_payment_cart_valid_promo_many_prducts" name="test_payment_cart_valid_promo_many_prducts" metainfo="test_payment_cart_valid_promo_many_prducts" status="passed">
                    <output type="stdout">PASSED [ 75%]
</output>
                </test>
                <test duration="15413" locationUrl="python&lt;C:\Users\Qasquad1\PycharmProjects\lime-qa\tests&gt;://tests.test_smoke.TestSmoke.test_burger_menu_check_products" name="test_burger_menu_check_products" metainfo="test_burger_menu_check_products" status="failed">
                    <output type="stdout">FAILED         [ 77%]
</output>
                    <output type="stderr">tests\test_smoke.py:215 (TestSmoke.test_burger_menu_check_products)..\venv\lib\site-packages\selene\core\wait.py:102: in for_
    return fn(self._entity)
..\venv\lib\site-packages\selene\core\condition.py:136: in __call__
    return self._fn(*args, **kwargs)
..\venv\lib\site-packages\selene\core\condition.py:74: in fn
    raise ConditionNotMatchedError()  # todo: try to handle printing actual values here too...
E   selene.core.exceptions.ConditionNotMatchedError: condition not matched
During handling of the above exception, another exception occurred:
test_smoke.py:223: in test_burger_menu_check_products
    page.wait_element_hidden(page.block_catalog)
..\pages\base.py:71: in wait_element_hidden
    element.assure(hidden, 15)
..\venv\lib\site-packages\selene\core\entity.py:627: in assure
    return self.should(condition, timeout)
..\venv\lib\site-packages\selene\core\entity.py:538: in should
    return self.with_(Config(timeout=timeout)).should(condition)
..\venv\lib\site-packages\selene\core\entity.py:540: in should
    super().should(condition)
..\venv\lib\site-packages\selene\core\entity.py:127: in should
    self.wait.for_(condition)
..\venv\lib\site-packages\selene\core\wait.py:125: in for_
    raise self._hook_failure(failure)
E   selene.core.exceptions.TimeoutException: Message: 
E   
E   Timed out after 15s, while waiting for:
E   browser.element(('xpath', "//nav[@class='mainmenu']")).is hidden
E   
E   Reason: ConditionNotMatchedError: condition not matched
E   Screenshot: file://C:\Users\Qasquad1\.selene\screenshots\1690257105839\1690257105844.png
E   PageSource: file://C:\Users\Qasquad1\.selene\screenshots\1690257105839\1690257105844.html
</output>
                </test>
                <test duration="46955" locationUrl="python&lt;C:\Users\Qasquad1\PycharmProjects\lime-qa\tests&gt;://tests.test_smoke.TestSmoke.test_burger_menu" name="test_burger_menu" metainfo="test_burger_menu" status="failed">
                    <output type="stdout">FAILED                        [ 79%]
</output>
                    <output type="stderr">tests\test_smoke.py:115 (TestSmoke.test_burger_menu)..\venv\lib\site-packages\selene\core\wait.py:102: in for_
    return fn(self._entity)
..\venv\lib\site-packages\selene\core\condition.py:136: in __call__
    return self._fn(*args, **kwargs)
..\venv\lib\site-packages\selene\core\condition.py:91: in fn
    if not predicate(entity):
..\venv\lib\site-packages\selene\core\match.py:37: in &lt;lambda&gt;
    'is visible', lambda element: element().is_displayed()
..\venv\lib\site-packages\selene\core\entity.py:179: in __call__
    return self._locator()
..\venv\lib\site-packages\selene\core\locator.py:34: in __call__
    return self._locate()
..\venv\lib\site-packages\selene\core\entity.py:1529: in &lt;lambda&gt;
    f'{self}.element({by})', lambda: self.driver.find_element(*by)
..\venv\lib\site-packages\selenium\webdriver\remote\webdriver.py:976: in find_element
    return self.execute(Command.FIND_ELEMENT, {
..\venv\lib\site-packages\selenium\webdriver\remote\webdriver.py:321: in execute
    self.error_handler.check_response(response)
..\venv\lib\site-packages\selenium\webdriver\remote\errorhandler.py:242: in check_response
    raise exception_class(message, screen, stacktrace)
E   selenium.common.exceptions.NoSuchElementException: Message: no such element: Unable to locate element: {"method":"xpath","selector":"//span[@title = 'Ваша корзина пуста']"}
E     (Session info: chrome=114.0.5735.248)
During handling of the above exception, another exception occurred:
test_smoke.py:150: in test_burger_menu
    page.wait_element_assure(page.cart)
..\pages\base.py:65: in wait_element_assure
    element.assure(visible, 15)
..\venv\lib\site-packages\selene\core\entity.py:627: in assure
    return self.should(condition, timeout)
..\venv\lib\site-packages\selene\core\entity.py:538: in should
    return self.with_(Config(timeout=timeout)).should(condition)
..\venv\lib\site-packages\selene\core\entity.py:540: in should
    super().should(condition)
..\venv\lib\site-packages\selene\core\entity.py:127: in should
    self.wait.for_(condition)
..\venv\lib\site-packages\selene\core\wait.py:125: in for_
    raise self._hook_failure(failure)
E   selene.core.exceptions.TimeoutException: Message: 
E   
E   Timed out after 15s, while waiting for:
E   browser.element(('xpath', "//span[@title = 'Ваша корзина пуста']")).is visible
E   
E   Reason: NoSuchElementException: Message: no such element: Unable to locate element: {"method":"xpath","selector":"//span[@title = 'Ваша корзина пуста']"}
E     (Session info: chrome=114.0.5735.248)
E   
E   Screenshot: file://C:\Users\Qasquad1\.selene\screenshots\1690257105839\1690257105846.png
E   PageSource: file://C:\Users\Qasquad1\.selene\screenshots\1690257105839\1690257105846.html
</output>
                </test>
                <test duration="32005" locationUrl="python&lt;C:\Users\Qasquad1\PycharmProjects\lime-qa\tests&gt;://tests.test_smoke.TestSmoke.test_bonus_card_courier" name="test_bonus_card_courier" metainfo="test_bonus_card_courier" status="passed">
                    <output type="stdout">PASSED                 [ 81%]
</output>
                </test>
                <test duration="27609" locationUrl="python&lt;C:\Users\Qasquad1\PycharmProjects\lime-qa\tests&gt;://tests.test_smoke.TestSmoke.test_first_login" name="test_first_login" metainfo="test_first_login" status="failed">
                    <output type="stdout">FAILED                        [ 83%]
</output>
                    <output type="stderr">tests\test_smoke.py:455 (TestSmoke.test_first_login)..\venv\lib\site-packages\selene\core\wait.py:102: in for_
    return fn(self._entity)
..\venv\lib\site-packages\selene\core\condition.py:136: in __call__
    return self._fn(*args, **kwargs)
..\venv\lib\site-packages\selene\core\condition.py:91: in fn
    if not predicate(entity):
..\venv\lib\site-packages\selene\core\match.py:37: in &lt;lambda&gt;
    'is visible', lambda element: element().is_displayed()
..\venv\lib\site-packages\selene\core\entity.py:179: in __call__
    return self._locator()
..\venv\lib\site-packages\selene\core\locator.py:34: in __call__
    return self._locate()
..\venv\lib\site-packages\selene\core\entity.py:1529: in &lt;lambda&gt;
    f'{self}.element({by})', lambda: self.driver.find_element(*by)
..\venv\lib\site-packages\selenium\webdriver\remote\webdriver.py:976: in find_element
    return self.execute(Command.FIND_ELEMENT, {
..\venv\lib\site-packages\selenium\webdriver\remote\webdriver.py:321: in execute
    self.error_handler.check_response(response)
..\venv\lib\site-packages\selenium\webdriver\remote\errorhandler.py:242: in check_response
    raise exception_class(message, screen, stacktrace)
E   selenium.common.exceptions.NoSuchElementException: Message: no such element: Unable to locate element: {"method":"xpath","selector":"//button[contains(.,"Выйти")]"}
E     (Session info: chrome=114.0.5735.248)
During handling of the above exception, another exception occurred:
test_smoke.py:461: in test_first_login
    page = SmokePage()
..\pages\smoke.py:602: in user_first_lk
    page.check_logout_btn_is_visible()
..\pages\login.py:42: in check_logout_btn_is_visible
    self.wait_element_assure(self.logout_btn)
..\pages\base.py:65: in wait_element_assure
    element.assure(visible, 15)
..\venv\lib\site-packages\selene\core\entity.py:627: in assure
    return self.should(condition, timeout)
..\venv\lib\site-packages\selene\core\entity.py:538: in should
    return self.with_(Config(timeout=timeout)).should(condition)
..\venv\lib\site-packages\selene\core\entity.py:540: in should
    super().should(condition)
..\venv\lib\site-packages\selene\core\entity.py:127: in should
    self.wait.for_(condition)
..\venv\lib\site-packages\selene\core\wait.py:125: in for_
    raise self._hook_failure(failure)
E   selene.core.exceptions.TimeoutException: Message: 
E   
E   Timed out after 15s, while waiting for:
E   browser.element(('xpath', '//button[contains(.,"Выйти")]')).is visible
E   
E   Reason: NoSuchElementException: Message: no such element: Unable to locate element: {"method":"xpath","selector":"//button[contains(.,"Выйти")]"}
E     (Session info: chrome=114.0.5735.248)
E   
E   Screenshot: file://C:\Users\Qasquad1\.selene\screenshots\1690257105839\1690257105847.png
E   PageSource: file://C:\Users\Qasquad1\.selene\screenshots\1690257105839\1690257105847.html
</output>
                </test>
                <test duration="25869" locationUrl="python&lt;C:\Users\Qasquad1\PycharmProjects\lime-qa\tests&gt;://tests.test_smoke.TestSmoke.test_cart_del_product_user" name="test_cart_del_product_user" metainfo="test_cart_del_product_user" status="passed">
                    <output type="stdout">PASSED              [ 85%]
</output>
                </test>
                <test duration="38734" locationUrl="python&lt;C:\Users\Qasquad1\PycharmProjects\lime-qa\tests&gt;://tests.test_smoke.TestSmoke.test_card_self" name="test_card_self" metainfo="test_card_self" status="passed">
                    <output type="stdout">PASSED                          [ 87%]
</output>
                </test>
                <test duration="26210" locationUrl="python&lt;C:\Users\Qasquad1\PycharmProjects\lime-qa\tests&gt;://tests.test_smoke.TestSmoke.test_pay_bonus_card_not_valid_pick_point" name="test_pay_bonus_card_not_valid_pick_point" metainfo="test_pay_bonus_card_not_valid_pick_point" status="passed">
                    <output type="stdout">PASSED [ 89%]
</output>
                </test>
                <test duration="686" locationUrl="python&lt;C:\Users\Qasquad1\PycharmProjects\lime-qa\tests&gt;://tests.test_smoke.TestSmoke.test_login_screen" name="test_login_screen" metainfo="test_login_screen" status="passed">
                    <output type="stdout">PASSED                       [ 91%]
</output>
                </test>
                <test duration="35281" locationUrl="python&lt;C:\Users\Qasquad1\PycharmProjects\lime-qa\tests&gt;://tests.test_smoke.TestSmoke.test_bonus_card_self" name="test_bonus_card_self" metainfo="test_bonus_card_self" status="passed">
                    <output type="stdout">PASSED                    [ 93%]
</output>
                </test>
                <test duration="24323" locationUrl="python&lt;C:\Users\Qasquad1\PycharmProjects\lime-qa\tests&gt;://tests.test_smoke.TestSmoke.test_payment_promocodes_6000_sum_6000" name="test_payment_promocodes_6000_sum_6000" metainfo="test_payment_promocodes_6000_sum_6000" status="passed">
                    <output type="stdout">PASSED   [ 95%]
</output>
                </test>
                <test duration="34064" locationUrl="python&lt;C:\Users\Qasquad1\PycharmProjects\lime-qa\tests&gt;://tests.test_smoke.TestSmoke.test_card_self_no_many" name="test_card_self_no_many" metainfo="test_card_self_no_many" status="passed">
                    <output type="stdout">PASSED                  [ 97%]
</output>
                </test>
                <test duration="10903" locationUrl="python&lt;C:\Users\Qasquad1\PycharmProjects\lime-qa\tests&gt;://tests.test_smoke.TestSmoke.test_registration_negative" name="test_registration_negative" metainfo="test_registration_negative" status="passed">
                    <output type="stdout">PASSED              [100%]
</output>
                </test>
            </suite>
        </suite>
    </suite>
</testrun>
